ts/components/UserDetailsDialog.tsx

```diff
@@ -46,6 +48,11 @@ export class UserDetailsDialog extends React.Component<Props> {
             {cancelText}
           </button>
 
+          <SessionToast
+            title="This is a notification!"
+            type={SessionToastType.Error}
+          />
```

Comment:
Is not this a test which should be removed?

Reply:
Yes, removing.